engine-comparison/begin-experiment.sh

```diff
@@ -18,7 +18,7 @@ INSTANCE_NAME="dispatcher-${DD}-${MM}"
 # echo "Restarting gcloud instance. Instance should already be created (with gcloud_creator.sh)"
 create_or_start $INSTANCE_NAME
 robust_begin_gcloud_ssh $INSTANCE_NAME
-gcloud compute ssh $INSTANCE_NAME --command="mkdir ~/input"
+gcloud compute ssh $INSTANCE_NAME --command="rm -fr ~/input/fengine-configs && mkdir ~/input"
```

Comment:
The reason I don't delete the whole `~/input` is because, in theory, FTS could save much bandwidth if it weren't deleted. In practice, I think that scp copies every file anyways, so this isn't effective...

In general, this piece of code has been bugging me because, when run multiple times, everything is recopied even if it doesn't need to be (which likely 99% of code doesn't). I think that the best course of action is, as previously discussed, abandoning ssh&scp for startup scripts&google storage: google storage has `rsync`, which copies files more intelligently (only copies those which need to copied, transfers the deletion of a file between directories, etc)